packages/config/src/validation.ts

```diff
@@ -148,6 +163,33 @@ export const isOneOf = (...values: any[]): ((key: string, value: any) => ErrResu
   }
 }
 
+/**
+ * Checks if given array value for a key includes only members of the provided values.
+ * @example
+  ```
+  validate = v.isSubsetOf("foo", "bar", "baz")
+  validate("example", ["foo"]) // true
+  validate("example", ["bar", "baz"]) // true
+  validate("example", ["foo", "else"]) // error message string
+  validate("example", ["foo", "bar", "baz", "else"]) // error message string
+  ```
+  */
+export const isSubsetOf = (...values: any[]): ((key: string, value: any) => ErrResult | true) => {
+  const validValues = values.map((a) => str(a)).join(', ')
+
+  return (key, value) => {
+    if (!Array.isArray(value)) {
```

Comment:
Does it make sense for these to return the same error message if we were expecting an array of configuration? It might help guide the user a bit more accurately. Otherwise, we can just condense the logic into one statement and remove the other `if` block.
```suggestion
    if (!Array.isArray(value) || !value.every((v) => values.includes(v))) {
```